src/main/java/staffconnect/logic/commands/SortCommand.java

```diff
@@ -0,0 +1,69 @@
+package staffconnect.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static staffconnect.logic.parser.CliSyntax.PREFIX_MODULE;
+import static staffconnect.logic.parser.CliSyntax.PREFIX_NAME;
+import static staffconnect.logic.parser.CliSyntax.PREFIX_PHONE;
+import static staffconnect.logic.parser.CliSyntax.PREFIX_VENUE;
+
+import java.util.Comparator;
+
+import staffconnect.commons.util.ToStringBuilder;
+import staffconnect.logic.Messages;
+import staffconnect.model.Model;
+import staffconnect.model.person.Person;
+
+/**
+ * Sorts and lists all persons in staff book in ascending order.
+ * Attribute shorthand matching is not case insensitive.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sort all persons by the attribute specified "
+            + "and displays them as a list with index numbers.\n"
+            + "Parameters: [ATTRIBUTE]\n"
+            + "[" + PREFIX_NAME + "] "
+            + "[" + PREFIX_PHONE + "] "
+            + "[" + PREFIX_MODULE + "] "
+            // + "[FACULTY] "
```

Comment:
I'll later add my part of codes on.

Reply:
I think I can update into this PR once "code is updated to used faculty" is merged